src/components/DayPickerRangeController.jsx

```diff
@@ -166,10 +166,17 @@ export default class DayPickerRangeController extends React.Component {
 
     const { currentMonth, visibleDays } = this.getStateForNewMonth(props);
 
+    // initialize phrases
```

Comment:
anything that depends on props in the constructor also must happen in `componentWillReceiveProps`.

Reply:
whoops, sorry, i saw your message.